modules/local/filter.nf

```diff
@@ -2,17 +2,18 @@ process FILTER {
     tag "$meta.id"
     label 'process_high'
 
-    conda "bioconda::seqkit=2.8.0"
+    conda "bioconda::seqkit=2.8.2"
```

Comment:
Ideally this should be moved into a `environment.yml` file and this module moved into a directory and renamed as `main.nf` for consistent with offiical modules, but not a bloker